generator/src/main/java/org/ovirt/sdk/go/WritersGenerator.java

```diff
@@ -252,8 +252,14 @@ else if (memberType instanceof StructType || memberType instanceof EnumType) {
         else if (memberType instanceof ListType) {
             ListType listType = (ListType) memberType;
             Type elementType = listType.getElementType();
-            String elementSingularTag = goNames.getTagStyleName(elementType.getName());
             if (elementType instanceof StructType || elementType instanceof EnumType) {
+                String elementSingularTag;
+                if(elementType instanceof StructType) {
+                    elementSingularTag = goNames.getTagStyleName(elementType.getName());
+                } else {
+                    elementSingularTag = schemaNames.getSchemaTagName(names.getSingular(member.getName()));
```

Comment:
```suggestion
                    elementSingularTag = goNames.getTagStyleName(names.getSingular(memberName));
```

Hi @mathianasj , one more tiny changes to keep code consistency. The function `goNames.getTagStyleName()` simply wraps the `schemaNames.getSchemaTagName()`, and `memberName` is already  defined before.
Thank you so much.

Reply:
@imjoey made these changes from suggestion